### PR TITLE
Replace obsolete devdocs links

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) replaces links to `https://devdocs.magento.com` topics that have already been migrated to the Adobe Devsite (`https://developer.adobe.com`).

## Affected pages

- See changed files

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My changes follow the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/sellers/extension-create.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ title: Create an extension
 
 Before you can submit your extension to Commerce Marketplace, the code must be complete, tested, and packaged. For technical information, see the Magento DevDocs for additional technical information for developing extensions and more:
 
--  [PHP Developer Guide](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/bk-extension-dev-guide.html)
--  [Best Practices for Extension Developers](https://devdocs.magento.com/guides/v2.4/ext-best-practices/bk-ext-best-practices.html)
--  [Magento Web APIs](https://devdocs.magento.com/guides/v2.4/get-started/bk-get-started-api.html)
--  [Frontend Developer Guide](https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/bk-frontend-dev-guide.html)
+-  [PHP Developer Guide](https://developer.adobe.com/commerce/php/development/)
+-  [Best Practices for Extension Developers](https://developer.adobe.com/commerce/php/best-practices/)
+-  [Magento Web APIs](https://developer.adobe.com/commerce/webapi/get-started/)
```
Comment (meker12) [inline src/pages/guides/sellers/extension-create.md:11]:
```suggestion
-  [Adobe Commerce and Magento Open Source Web APIs](https://developer.adobe.com/commerce/webapi/get-started/)
```

File: src/pages/guides/sellers/technical-reference.md [hunk 2]
```diff
@@ -6,26 +6,26 @@ title: Technical Reference
...
+-  [Register](https://developer.adobe.com/commerce/php/development/build/component-registration/)
+-  [Create](https://developer.adobe.com/commerce/php/development/build/)
+-  [Enable](https://developer.adobe.com/commerce/php/development/build/component-management/)
+-  [Test](https://developer.adobe.com/commerce/php/development/validate/test-component/)
+-  [Package](https://developer.adobe.com/commerce/php/development/package/component/)
+-  [Coding Standards](https://developer.adobe.com/commerce/php/coding-standards/)
+-  [Frontend Developers](https://developer.adobe.com/commerce/frontend-core/guide/)
+-  [Admin Pattern Library](https://developer.adobe.com/commerce/admin-developer/pattern-library/)
+-  [JavaScript Developers](https://developer.adobe.com/commerce/frontend-core/javascript/)
+-  [Magento Web APIs](https://developer.adobe.com/commerce/webapi/get-started/)
```
Comment (meker12) [inline src/pages/guides/sellers/technical-reference.md:23]:
```suggestion
-  [Adobe Commerce Web APIs](https://developer.adobe.com/commerce/webapi/get-started/)
```

**Review by meker12 — APPROVED** [review]
Two suggestions replacing "Magento" with "Adobe Commerce" -- otherwise ⭐

**Comment by jeff-matthews** [comment]
Thanks @meker12! I had auto-merge enabled, so the PR merged before I saw your suggestions. Please use the comment or request changes option when submitting your review if making suggestions.

**Pull request merged**